### PR TITLE
Put each chapters in its ownfile

### PR DESCRIPTION
**Pull request by rocky**
The file is getting large, so I'd like to split it up.  Splitting into chapters also helps a little when different people are working simultaneously on different chapters.

(Had also thought of pushing doc stuff down into its own folder, but one little thing at a time.)

Thoughts?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,5 @@
 /*.log
 /*.pdf
 /*.toc
+/changes.texi
```
Comment (pipcet) [inline .gitignore]: Why is changes.texi ignored? There's also a discrepancy between lines added and lines removed, I think because of this.

**Comment by rocky** [comment]
Yes you are correct. Is a mistake. Thanks. Travelling so I will handle
later.

**Comment by rocky** [comment]
@pipcet ping

**Comment by pipcet** [comment]
Oops, sorry, yes, this was fine!

**Pull request merged**